index.html

```diff
@@ -2582,179 +2582,141 @@ <h2>CBOR</h2>
+<li><a href="https://tools.ietf.org/id/draft-ietf-cbor-7049bis-12.html#section-3.2.2">Empty values</a> MUST be represented as a null literal (primitive type 22). </li>
+<li><a href="https://tools.ietf.org/id/draft-ietf-cbor-7049bis-12.html#section-3.2.2">Other values</a> MUST be represented as a String type. (major type 5)</li>
+<li><a href="https://tools.ietf.org/id/draft-ietf-cbor-7049bis-12.html#section-3.2.2">Indefinite-length items</a> MUST be made definite legth.</li>
+<li><a href="https://www.rfc-editor.org/rfc/rfc3339.txt">Datetime </a> MUST be encoded in UTF-8 (major type 3). This datetime value MUST be normalized to UTC 00:00, as indicated by the trailing "Z".</li>
+</ul>
+
+
+<p>Semantic interoperability with other core representations of the DID document is possible with additional constaints 
+  to the CBOR representation.  Additionally, for lossless conversion between other core representations a deterministic 
+  representation is desirable when producing DID Documents that are represented as CBOR, in addition to the suggestions
+  in section 3.9 of the CBOR [[RFC7049]] specification for deterministic mappings, 
+  the following constraints of the DID Document model MUST be followed:
+  These additional constaints include: </p>
+
+<ul>
+    <li>Property names MUST be represented as a byte string array (major type 2) and contain only UTF-8 strings. </li>
+    <li>All member properties of a CBOR encoded DID document MUST be either represented in the `@context`, be present in the 
+      <a href="https://www.w3.org/TR/did-spec-registries/">DID spec registries</a> or be explicitly documented as an extension of the DID document conformant to <a href="#extensibility"> § 4.3. Extensibility</a>.  </li>
+    <li>The byte string array's length in bytes is represented following the rules for positive integers (major type 0).  </li>
+    <li>Property names in each CBOR map MUST be unique. </li>
+    <li>Integer encoding MUST be as short as possible.</li>
+    <li>The expression of lengths in CBOR major types 2 through 5 MUST be as short as possible. </li>
+    <li>All floating point values MUST be encoded as 64-bits, even for integral values.</li>
+    <li>The keys in every map must be sorted lowest value to highest. Sorting is performed on the bytes of the representation of the keys. </li>
+
+  </ul>
+</p>
 
-        <ul>
-          <li>
-Map keys MUST be strings.
-          </li>
-          <li>
-Integer encoding MUST be as short as possible.
-          </li>
-          <li>
-The expression of lengths in CBOR major types 2 through 5 MUST be as short as
-possible.
-          </li>
-          <li>
-All floating point values MUST be encoded as 64-bits, even for integral values.
-          </li>
-        </ul>
+<p>
+  The property name `@context` MAY be used in the CBOR map and has the same semantic meaning as JSON-LD producers of a DID document in JSON representation.
```

Comment:
Strange for CBOR to talk about JSON-LD. While the statement is true, it should probably just stay silent on this item.

Reply:
I think this is in-line with the conclusion of last Thursday's special topic call.  `@context` MAY exist in `application/json` DID documents, NO json-ld processing is expected.  Same is true of the `application/cbor` DID document.

Reply:
It may be better to talk about what to do when consuming properties that are not defined in DID Core nor the registries, rather than specifically mentioning a particular property from another representation.